src/models/logging.ts

```diff
@@ -0,0 +1,14 @@
+import { knex } from '../db';
+
+type Logging = {
+    name : string
```

Comment:
I believe, we also need `created_at`, `updated_at` here

Reply:
I mean, define Model type and use it where we need some operation with the table. 
It can be `insert`, `select` and so on